### PR TITLE
Added Spinner to Join Button

### LINKED ISSUE #172: Adding loading Spinner to join button
As the user press the `join` button, to make the app user interface responsive, the presence of a loading spinner at the `join` button after the dialogue box will make the page transition experience better for the user.

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**What kind of change does this PR introduce?**

Improving the Experience of the Join an Organization Page

- The `CircularProgressIndicator` was added to the` join` button to improve the user experience 

**Did you add tests for your changes?**
Yes, the change was tested in Android systems and does not break anything.

**Summary**

The main motivation was to make the page transition better and improve the experience

This PR is related to Issue and Fixes #172

### CODE REVIEW

File: lib/views/pages/login_signup/login_page.dart [hunk 1]
```diff
@@ -417,7 +417,8 @@ class _LoginScreenState extends State<LoginPage> with TickerProviderStateMixin {
                     height: 20,
                   ),
                   FadeTransition(
-                    opacity: createAnimation,
+                    //changed opacity animation to match login button animation
```
Comment (Sagar2366) [inline lib/views/pages/login_signup/login_page.dart:420]: this change is already there in master

**Comment by Sagar2366** [comment]
@adithyaakrishna can you please share only lib/views/pages/organization/join_organization.dart file changes for this PR review that too add changes required to fix tissue.
Changes in other files are already added to master.

**Comment by adithyaakrishna** [comment]
@Sagar2366 I'm Closing this PR and Creating a New one with the changes you mentioned 😶

**Comment by Sagar2366** [comment]
> @Sagar2366 I'm Closing this PR and Creating a New one with the changes you mentioned 😶

@adithyaakrishna you could have just reverted the comment and continue working on this PR.

**Comment by adithyaakrishna** [comment]
> > @Sagar2366 I'm Closing this PR and Creating a New one with the changes you mentioned 😶
> 
> @adithyaakrishna you could have just reverted the comment and continue working on this PR.

I thought there was the same error like the prev one, my bad :/
For the issue #172 I created the PR #418 with that specific file.

**Pull request closed without merging**